lazy_tensor_core/test/test_extract_compiled_graph.py

```diff
@@ -50,6 +50,20 @@ def forward(self, a, b):
 #         b = torch.randn(2, 3, device="cpu") # eager device
 #         return a + b
 
+# The module was planned to cover the case that a Fx graph return an eager
+# tensor on the default device. It's harder than ModuleEagerTensor because
+# we can not just override the device argument to Lazy since there is no
+# explicit device argument.
+#
+# Unfortunately, the default fx tracer convert the return value of the forward
+# method to a constant.. Comment out for now
```

Comment:
how come FX didn't turn torch.tensor into a constant in yolov3 and hf_Bart?

Reply:
Because dynamo uses a customized tracer: https://github.com/facebookresearch/torchdynamo/blob/5b214ee9f64d4666e7ee0be240b370082200e97d/torchdynamo/output_graph.py#L49